src/services/completions.ts

```diff
@@ -712,7 +712,7 @@ namespace ts.Completions {
     function getFirstSymbolInChain(symbol: Symbol, enclosingDeclaration: Node, checker: TypeChecker): Symbol | undefined {
         const chain = checker.getAccessibleSymbolChain(symbol, enclosingDeclaration, /*meaning*/ SymbolFlags.All, /*useOnlyExternalAliasing*/ false);
         if (chain) return first(chain);
-        return isModuleSymbol(symbol.parent) ? symbol : symbol.parent && getFirstSymbolInChain(symbol.parent, enclosingDeclaration, checker);
+        return symbol.parent && (isModuleSymbol(symbol.parent) ? symbol : getFirstSymbolInChain(symbol.parent, enclosingDeclaration, checker));

```

Comment:
If `symbol.parent === undefined`, then we recursively call `getFirstSymbolInChain` with first argument `undefined`. Are you sure this won't just trigger an error that `we can't get `parent` from dotting off `undefined` in the next recursive call?

Does the test you added cover this case?

Reply:
Entire rest of the expression is in parentheses, meaning we do nothing if `symbol.parent === undefined`.